lego/apps/files/tests/test_api.py

```diff
@@ -44,3 +44,128 @@ def test_post_create_file_call(self, mock_create_file):
         except Exception:
             pass
         mock_create_file.assert_called_with(key, self.user, False)
+
+
+class SetSaveForUserTest(BaseAPITestCase):
+    fixtures = ["test_users.yaml", "test_files.yaml", "test_abakus_groups.yaml"]
+    key = "abakus.png"
+    token = "token"
+    url = f"/api/v1/files/{key}/set_save_for_use/"
```

Comment:
Use the `reverse` method instead of using url strings directly

Reply:
Also, if you could come up with a better endpoint for this that would be super nice. I don't think `set_save_for_use` is specifically meaningful or nice.